linq.d.ts

```diff
@@ -33,7 +33,7 @@ declare namespace Enumerable {
   export function unfold<T>(seed: T, func: (value: T) => T): IEnumerable<T>;
   export function defer<T>(enumerableFactory: () => IEnumerable<T>): IEnumerable<T>;
 
-  export interface IEnumerable<T> {
+  export interface IEnumerable<T> extends Iterable<T> {
```

Comment:
right, so, maybe I can add a reference to es2015.iterable somehow, so it works in es5.